plugins/eni/engine/dhclient.go

```diff
@@ -59,6 +59,9 @@ const (
 	// package and is not exposed anywhere. Hence, we declare this string to
 	// match the error message returned by the os package
 	processFinishedErrorMessage = "os: process already finished"
+
+	maxDHClientStartAttempts            = 10
+	durationBetweenDHClientStartRetries = 2 * time.Second
```

Comment:
Do we have some data for this, I'm a little worried about 20 seconds is not enough?

Reply:
Yeah, in my tests, i've seen p100 at `5s`.